website/src/app/[lang]/[region]/(website)/about-us/(sections)/landing-page.tsx

```diff
@@ -13,13 +13,94 @@ export default async function LandingPage({ lang }: { lang: WebsiteLanguage }) {
 
 	return (
 		<BaseContainer className="min-h-screen-navbar grid grid-cols-1 content-center items-center gap-16 md:grid-cols-5">
-			<p className="mx-auto max-w-lg md:col-span-3">
+			<div className="mx-auto max-w-lg space-y-5 md:col-span-3">
 				{translator.t<{ text: string; color?: FontColor }[]>('landing-page.title').map((title, index) => (
-					<Typography as="span" key={index} size="5xl" weight="bold" color={title.color}>
+					<Typography as="span" key={index} size="5xl" weight="bold" lineHeight="tight" color={title.color}>
 						{title.text}
 					</Typography>
 				))}
-			</p>
+				<Typography size="2xl" lineHeight="normal">
+					{translator.t('landing-page.subtitle')}
+				</Typography>
+				<div className="flex space-x-2 pt-5">
+					<div>
+						<Popover>
+							<PopoverTrigger>
+								<Badge
+									variant="outline"
+									className="hover:bg-muted-foreground text-muted-foreground hover:text-secondary-foreground transition-transform duration-300 hover:scale-105"
```

Comment:
I think it's not so nice to have these configurations here in `className`. They should go into the Badge component so that it behaves the same every time we use it.